python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -146,83 +152,85 @@ def rearrange_by_hash(
     groups = []
     splits = []
     combines = []
+    inputs = []
 
-    inputs = [
-        tuple(digit(i, j, k) for j in range(stages))
-        for i in range(k ** stages)
-    ]
+    start = {}
+    end = {}
 
     token = tokenize(df, columns, max_branch)
+    shuffle_combine_token = "shuffle-combine-" + token
```

Comment:
I mentioned this in a separate PR, but I believe dask covention is to use "`_name`" rather than "`_token`" for these task labels - However, I don't feel strongly one way or the other (So, only make this change if you want).